Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: build generate image clean
+
+CGO_ENABLED=0
+
+all: build
+
+build: generate
+	@go build -tags "netgo static_build" -installsuffix netgo -ldflags -w .
+
+generate:
+	@go generate
+
+image:
+	@docker build -t goproxyio/goproxy .
```

Comment:
@hxzhao527 is right, we are not maintenance `goproxyio/goproxy` org on docker hub now, `goproxy/goproxy` is better.

Reply:
This target is just for local testing anyway; But I'll amend the image name. Docker Hub images should be automated (//I hope//)